modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -1403,10 +1473,26 @@ Status description:: error: system configuration or operation exception - bolt i
 
 Status description:: error: system configuration or operation exception - unsupported operation of a sharded database. Feature: `{ $feat }` is not available in a sharded database.
 
+=== 51N72
+
+Status description:: error: system configuration or operation exception - memory pool out of memory. Failed to allocate memory in a memory pool. See `{ $cfgSetting }` in the neo4j configuration.
```

Comment:
```suggestion
Status description:: error: system configuration or operation exception - memory pool out of memory. Failed to allocate memory in a memory pool. See `{ $cfgSetting }` in the neo4j.conf file.
```

I think this and the two below were changed based on PR feedback after you made this PR.